packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,257 @@
+import {
+  StackLayout,
+  Avatar,
+  Text,
+  Card,
+  Button,
+  Tooltip,
+  FlexLayout,
+} from "@salt-ds/core";
+import {
+  WomanIcon,
+  MessageIcon,
+  LocationIcon,
+  CallIcon,
+  ChatIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+} from "@salt-ds/icons";
+import { Meta } from "@storybook/react";
+import { useState } from "react";
+
+export default {
+  title: "Patterns/Contact Details",
+} as Meta;
+
+const basicContact = {
+  primary: "Alex Brailescu",
+  secondary: "Blackrock Advisors (UK) Limited",
+  metadata: [
+    {
+      label: "Role",
+      value: "Analyst",
+      icon: <WomanIcon />,
+    },
+    {
+      label: "Location",
+      value: "London, GBR",
+      icon: <LocationIcon />,
+    },
+    {
+      label: "Office",
+      value: "+44 2077 431102",
+      icon: <CallIcon />,
+    },
+    {
+      label: "Email",
+      value: "alex.brailescu@blackrock.com",
+      icon: <MessageIcon />,
+    },
+  ],
+};
+
+export const ContactDetails = ({
+  avatarSrc = "https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png",
+}) => {
+  return (
+    <StackLayout direction={"column"} gap={1}>
+      <Avatar
+        src={avatarSrc}
+        aria-label={basicContact.primary}
+        fallbackIcon={<WomanIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3">{basicContact.primary}</Text>
+        <Text styleAs="h4">{basicContact.secondary}</Text>
+        <StackLayout direction={"row"}>
+          <StackLayout gap={1} direction={"column"}>
+            {basicContact.metadata.map((metadata) => {
+              return (
+                <Text styleAs="label" key={metadata.label}>
```

Comment:
looks like label is 11px whereas uitk is 12px font size

Reply:
as in, this corresponding text on uitk

Reply:
This is a valid change, patterns should not have overrides to foundational values e.g. font-size, line-height

Reply:
but does it need to use styleAs=label? I think if it is left as default Text it will be body which is 12px

Reply:
My original comment should really be worded more like - do we need to use `styleAs="label"` here? It looks like the uitk contact detail is using body